src/createNumbersGenerator.js

```diff
@@ -26,7 +26,26 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let filteredNumbers = [];
+  if (min === undefined && max === undefined && numbers !== undefined) {
+    filteredNumbers = numbers.slice();
+  } else if (numbers !== undefined) {
+    for (const item of numbers) {
+      if (item >= min && item <= max) {
```

Comment:
good, but if we will receive only `max` or `min`?

you can solve this problem with another check or add default values to min and max